package.json

```diff
@@ -75,57 +75,57 @@
     "zustand": "^4.5.5"
   },
   "devDependencies": {
-    "@ianvs/prettier-plugin-sort-imports": "^4.3.1",
+    "@ianvs/prettier-plugin-sort-imports": "^4.4.0",
     "@mswjs/http-middleware": "^0.10.2",
     "@oxide/openapi-gen-ts": "~0.5.0",
     "@playwright/test": "^1.49.1",
     "@testing-library/dom": "^10.4.0",
-    "@testing-library/jest-dom": "^6.5.0",
-    "@testing-library/react": "^16.0.1",
+    "@testing-library/jest-dom": "^6.6.3",
+    "@testing-library/react": "^16.1.0",
     "@types/lodash.throttle": "^4.1.9",
     "@types/md5": "^2.3.5",
     "@types/mousetrap": "^1.6.15",
-    "@types/react": "^18.3.5",
-    "@types/react-dom": "^18.3.0",
-    "@types/react-is": "^18.3.0",
+    "@types/react": "^18.3.18",
+    "@types/react-dom": "^18.3.5",
+    "@types/react-is": "^18.3.1",
     "@types/semver": "^7.5.8",
     "@types/uuid": "^10.0.0",
     "@typescript-eslint/eslint-plugin": "~8.5.0",
     "@typescript-eslint/parser": "~8.5.0",
     "@vitejs/plugin-basic-ssl": "^1.2.0",
     "@vitejs/plugin-react-swc": "^3.7.2",
     "autoprefixer": "^10.4.20",
-    "eslint": "^8.57.0",
+    "eslint": "^8.57.1",
     "eslint-config-prettier": "^9.1.0",
-    "eslint-plugin-import": "^2.30.0",
-    "eslint-plugin-jsx-a11y": "^6.10.0",
-    "eslint-plugin-playwright": "^1.6.2",
+    "eslint-plugin-import": "^2.31.0",
+    "eslint-plugin-jsx-a11y": "^6.10.2",
+    "eslint-plugin-playwright": "^1.8.3",
     "eslint-plugin-prettier": "^5.2.1",
-    "eslint-plugin-react": "^7.36.1",
+    "eslint-plugin-react": "^7.37.3",
     "eslint-plugin-react-hook-form": "^0.3.0",
     "eslint-plugin-react-hooks": "^4.6.2",
-    "husky": "^9.1.6",
+    "husky": "^9.1.7",
     "identity-obj-proxy": "^3.0.0",
     "ip-num": "^1.5.1",
-    "jsdom": "^25.0.0",
-    "msw": "^2.6.2",
-    "oxlint": "^0.14.0",
+    "jsdom": "^25.0.1",
+    "msw": "^2.7.0",
+    "oxlint": "^0.14.1",
     "patch-package": "^8.0.0",
-    "postcss": "^8.4.35",
+    "postcss": "^8.4.49",
     "postcss-import": "^16.1.0",
-    "postcss-nesting": "^13.0.0",
+    "postcss-nesting": "^13.0.1",
     "postcss-pseudo-classes": "^0.4.0",
-    "prettier": "^3.3.3",
-    "prettier-plugin-tailwindcss": "^0.6.6",
+    "prettier": "^3.4.2",
+    "prettier-plugin-tailwindcss": "^0.6.9",
     "resize-observer-polyfill": "^1.5.1",
     "semver": "^7.6.3",
-    "tailwindcss": "^3.4.11",
-    "tsx": "^4.19.1",
-    "type-fest": "^4.26.1",
+    "tailwindcss": "^3.4.17",
+    "tsx": "^4.19.2",
+    "type-fest": "^4.31.0",
     "typescript": "~5.7.0",
-    "vite": "^6.0.3",
+    "vite": "^6.0.7",
     "vite-plugin-html": "^3.2.2",
-    "vite-tsconfig-paths": "^5.1.3",
+    "vite-tsconfig-paths": "^5.1.4",
     "vitest": "^3.0.0-beta.2"
```

Comment:
this doesn't come up automatically because it's not semver, but it looks like you can bump this to beta 4
https://github.com/vitest-dev/vitest/releases/tag/v3.0.0-beta.4